DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
     if not processFiles["OK"]:
       log.error( processFiles["Message"] )
       return processFiles
-    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
+    processFiles = processFiles['Value']
+    if processFiles['toRegister']:
+      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
```

Comment:
The Magic DB is the SystemLoggingDB where all the entries are classified by the static message part which is the first argument. The variable part is printed in the log, so you can print the number of files there. Make it so, that it makes sense for the reader and does not screw up SystemLogging at the same time.